src/screens/WebView/HelpCenter.js

```diff
@@ -31,7 +32,7 @@ const styles = StyleSheet.create({
   container: {
     // paddingHorizontal: 20,
     flex: 1,
-    backgroundColor: '#fff'
+    backgroundColor: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded color value to using a constant from a centralized theme file is a good practice. It improves maintainability and consistency across the application.
```diff
-    backgroundColor: '#fff'
+    backgroundColor: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->